core/templates/dev/head/css/oppia.css

```diff
@@ -2126,18 +2116,27 @@ pre.oppia-pre-wrapped-text {
     the same line-height, margin-top and margin-bottom attributes)
 */
 .oppia-state-content-display-html > p,
+.oppia-feedback-display-html > p,
```

Comment:
Do you know if there's a way to handle this without having to add a new selector for each field type? Otherwise we risk missing a bunch of extra things, e.g. multiple choice input options might also admit paragraphs.

For example, I was wondering if we could just have two selectors (one for the "RTE editor" and one for the "RTE viewer") instead -- maybe it might be worth introducing a common "RTE viewer" directive.

Reply:
Yeah I think that sounds like a good idea, I'll file it as a separate issue.

Reply:
OK, thanks! Feel free to merge this once you've done that.